### DISCUSSION TITLE
Allow narrower vertical taskbar

### DISCUSSION THREAD

**Discussion by Infi220** [valinet/ExplorerPatcher#1569] [Ideas]
Please, allow users to make the taskbar narrower, ideally 32 pixels. In Win 10 I'm able to do that with the help of "7+ Taskbar Tweaker".  This also hides the icon labels and saves a lot of screen space. See below:

![taskbar](https://github.com/valinet/ExplorerPatcher/assets/34073985/a8582733-a315-47f8-b406-4099edf7ed39)

**Comment by gfmoore** [comment]
Yes please for me too :)

Thanks for all your efforts, this is marvellous :)

**Comment by pyrates999** [comment]
Since you can make the change using 7+ taskbar tweaker, EP doesn't need to add this.

**Comment by Amrsatrio** [comment]
I accidentally added this when developing the reimplemented 10 taskbar... maybe later then.

**Reply by Infi220** [reply]
Do you mean you had a fix for it? If so, could you please enable it?

**Comment by Infi220** [comment]
Can we please have this implemented to allow a narrow vertical taskbar of 32 pixels?
The vertical taskbar is way too wide at 62+ pixels, and 7+TT no longer works with Explorer Patcher.
Actual:
<img width="110" height="268" alt="image" src="https://github.com/user-attachments/assets/47f42bb1-3814-45eb-b8d8-b52ed300301e" />

Expected:
<img width="94" height="268" alt="image" src="https://github.com/user-attachments/assets/7bf109c0-9fbd-4125-b5fc-f3ce176d7287" />

**Reply by pyrates999** [reply]
Have you tried this?  Right click on the taskbar and untick `Lock the taskbar`.  Then you can resize the taskbar.  Then right click on the taskbar and tick `Lock the taskbar`.

**Reply by Infi220** [reply]
Yes, that's exactly what I'm doing, but the smallest width that EP allows is 62 pixels, which is unnecessarily wide. 
When 7+TT was still compatible with EP, it allowed 32 pixels minimum width.

**Comment by Infi220** [comment]
Is there any hope of having it enabled, or is there any workaround?

**Reply by pyrates999** [reply]
try posting in the windhawk mods github as a request for a new mod.  And include that 7+ TT had it implemented before for the windows 10 taskbar.

**Reply by Infi220** [reply]
The problem with windhawk is that it's a 700MB installation. It would be much better to have it simply enabled in EP. @Amrsatrio said he already "accidentally added it" in 2024.

**Reply by pyrates999** [reply]
You don't have much of a choice.  See here: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation#why-is-7-taskbar-tweaker-not-compatible-with-this

All the features that 7+ TT provided are being migrated into windhawk as mods.
